### PR TITLE
doc: tm_startswith and tm_endswith.

### PR DESCRIPTION
**Pull request by i4ki**
As this come from opentofu, the documentation was mostly copied from their docs.

### CODE REVIEW

File: cli/reference/functions/tm_endswith.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: tm_endswith | Functions | Configuration Language
+description: The `tm_endswith` returns true if the first string ends with that exact suffix.
+---
+
+# `tm_endswith` Function
+
+`tm_endswith` takes two values: a string to check and a suffix string. The function returns true if the first string ends with that exact suffix.
```
Comment (AnnuCode) [inline cli/reference/functions/tm_endswith.md:8]:
```suggestion
The `tm_endswith` function checks if a given string ends with a specified suffix. It accepts two arguments: the string to evaluate and the suffix string. The function returns true if the string ends with the exact suffix provided.
```

File: cli/reference/functions/tm_endswith.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+tm_endswith("hello world", "world")
+true
+
+tm_endswith("hello world", "hello")
+false
+```
+
+## Related Functions
+
+* [`tm_startswith`](./tm_startswith.md) takes two values: a string to check and a prefix string. The function returns true if the string begins with that exact prefix.
```
Comment (AnnuCode) [inline cli/reference/functions/tm_endswith.md:26]:
```suggestion
* The [tm_startswith](./tm_startswith.md) function checks if a given string starts with a specified prefix. It accepts two arguments: the string to evaluate and the prefix string. The function returns true if the string begins with the exact prefix provided.
```

File: cli/reference/functions/tm_startswith.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+tm_startswith("hello world", "hello")
+true
+
+startswith("hello world", "world")
+false
+```
+
+## Related Functions
+
+* [`tm_endswith`](./tm_endswith.md) takes two values: a string to check and a suffix string. The function returns true if the first string ends with that exact suffix.
```
Comment (AnnuCode) [inline cli/reference/functions/tm_startswith.md:26]:
```suggestion
* The `tm_endswith` function checks if a given string ends with a specified suffix. It accepts two arguments: the string to evaluate and the suffix string. The function returns true if the string ends with the exact suffix provided.
```

**Review by soerenmartius — APPROVED** [review]
LGTM, @i4ki this needs to be added to the sidebar also

**Pull request merged**